src/components/Table/Table.test.jsx

```diff
@@ -567,54 +705,6 @@ describe('Table', () => {
     unmount();
   });
 
-  it('cells should wrap (not truncate) for wrapCellText:auto + resize + table-layout:auto', () => {
```

Comment:
Why not just leave in all the previous tests, especially since they are verifying using an alternate methodology.

Reply:
I wanted to make the tests more resilient and not rely on checking for props, especially for the internal components. Since the props wrapText && truncateCellText would be overlapping and confusing after this PR I wanted to merge them and reduce the table code complexity, which meant that the tests relying on props and not HTML outcome would have to be refactored.

Reply:
I get the logic, but, the existing unit tests are in place to protect against regressions.  If you having to remove tests and completely rewrite them as a result of a change, then that tells me you are hiding a regression.  Leaving the existing tests in place and then create a separate story LATER to remove what you considered redundant tests makes more sense, since, those existing tests will help you catch regressions, but they can't help if you create new tests that you think are 1-1 tests of the original, and erase the ones.  I'm a little sensitive here because I've seen a number of cases in table/dashboard where tests were erased as part of a change and then I'm left dealing with finding those when I'm taking an update.

Reply:
So in this particular case it is basically a trade off between a) introducing more complexity to the table when there is the option to reduce it and b) accepting the risk and rewriting the existing suboptimal tests to allow us the possibility to refactor the table and improve our future test situation.

Reply:
But you are not accepting any risk, are you creating it :)  You are asking me to accept your risk, and given that I'm now closing out 3 weeks effort to move to the latest Carbon/IoT and I'll be pushing into 4 weeks of that effort before it is done... I'm not sure I like it when you guys create "acceptable" risk :(

Reply:
Language confusion, I meant "accepting the risk with rewriting the existing... ". I believe we all share the risk since our goal is always delivering changes without introducing regressions. But to ease your worry I have created a new PR that does not contain the code refactoring, hence keeping the old unit tests. I will close this PR and we will add the refactoring in a separate Issue for a major release where we also remove the export of TableHead and TableBody.

New PR is available here
https://github.com/carbon-design-system/carbon-addons-iot-react/pull/1651